guides/cookbook/index.md

```diff
@@ -1,3 +1,4 @@
 # Cookbook
 
 - [Incremental Adoption Guide](./incremental-adoption-guide.md)
+- [Model Name: singular or plural? What to choose? Why is that?](./model-name.md)
```

Comment:
```suggestion
- [Naming Conventions: Should resource types be singular or plural? What to choose? Why is that?](./model-name.md)
```

I'd suggest we also call the file `naming-conventions.md`. That way we can also add a bit on `snake_case` vs `camelCase` key conventions etc. if desired.